client/me/privacy/controller.js

```diff
@@ -0,0 +1,23 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import PrivacyComponent from 'me/privacy/main';
+
+export default function privacyController( context, next ) {
+	if ( ! config.isEnabled( 'me/privacy' ) ) {
+		page.redirect( '/me' );
```

Comment:
Stylistically we prefer early returns if possible. This reduces the need for if/else blocks.

So this may look like:
```js
if ( ! config.isEnabled( 'me/privacy' ) ) {
    return page.redirect( '/me' );
}
context.primary = React.createElement( PrivacyComponent );
next();
```